packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -65,36 +57,16 @@ const ConfirmResetPassword = ({
     navigateToPasswordReset();
   }
 
-  useInterval(async () => {
-    try {
-      // A bit unconventional but this endpoint will throw an invalid token error
-      // that represents the password has been reset (or that the token is expired)
-      const isValid = await account.resetPasswordStatus(
-        currentPasswordForgotToken
-      );
-      if (!isValid) {
-        hardNavigateToContentServer('/signin');
-      } else {
-        // TODO: Not sure about this. It works with the flow... but.
-        setOriginalTabMarker();
-      }
-    } catch (err) {
-      setIsPolling(null);
-    }
-  }, isPolling);
-
   const resendEmailHandler = async () => {
     try {
       if (isOAuthIntegration(integration)) {
-        const result = await account.resetPassword(
+        await account.resetPassword(
           email,
           integration.getService(),
           integration.getRedirectUri()
         );
-        setCurrentPasswordForgotToken(result.passwordForgotToken);
```

Comment:
A bit on the fence about this one. Is it possible to turn polling off before resending the reset email, then turn it back on after?

Seems like there is some timing issue once it `setCurrentPasswordForgotToken` and then runs the polling function.

Reply:
🤩 Elegant! I like that.

Reply:
@vbudhram I've updated the PR to disable the polling while the link is resent (and re-enable once complete). Could you take a peek?